packages/editor/src/components/post-sticky/check.js

```diff
@@ -26,11 +21,7 @@ export default compose( [
 	withSelect( ( select ) => {
 		const post = select( editorStore ).getCurrentPost();
 		return {
-			hasStickyAction: get(
-				post,
-				[ '_links', 'wp:action-sticky' ],
-				false
-			),
+			hasStickyAction: post._links?.[ 'wp:action-sticky' ],
```

Comment:
`getCurrentPost()` will always be an object:

https://github.com/WordPress/gutenberg/blob/4944315cac156c78313121657dff9d58ce8f4d3f/packages/editor/src/store/selectors.js#L170